client/src/GlobalStyle.js

```diff
@@ -0,0 +1,43 @@
+import { createGlobalStyle } from "styled-components/macro";
+
+const GlobalStyle = createGlobalStyle`
+
+*,*::before,*::after{
+  box-sizing:border-box
+}
+
+:root {
+--gradient-color: linear-gradient(#F2D3AC, #FFFFFF );
+--active-color: #037F8C;
+--primary-color: #F8E4CF;
+--text-color: #262626
+}
+body {
+font-family: 'Encode Sans', sans-serif;
+-webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+font-size: 16px;
+font-weight: 300;
+background: var(--gradient-color);
+margin: 0;
+color: var(---text-color);
+height: 100vh
+
+}
+h1{
+font-size: 24px;
+font-weight: 500;
```

Comment:
I`d declare font sizes in rem rather than px :)

Reply:
thanks 👍🏻